documentation/principles/empowering.md

```diff
@@ -2,12 +2,13 @@
 layout: sub-navigation
 order: 106
 title: Empowering
-excerpt: Short extract
+excerpt: We aim to allow teams to confidently deliver at pace and scale.
 tags:
 - principle
 eleventyNavigation:
   parent: principles
 ---
-Core Cloud aims to be light on process and empowering for teams.\
-Core Cloud allows team to engage at the right level for them; teams can create and manage their own resources or make use of our support model.\
-Core Cloud aims for 'Minimum Viable Platform' with no unnecessary abstractions or coupling.
\ No newline at end of file
+**We design for self-service.** \
+Teams should be able to provision their own resources from our service catalogue or make use of our support model and can pick which is right for them.
+
+**We aim to be a 'Minimum Viable Platform'** - light on process, with no unnecessary abstractions or coupling.
```

Comment:
```suggestion
We endeavour to deliver the right balance of governance and freedom to ensure that we can deliver our guarantee of a secure platform to operate on, without stifling your freedom to innovate
```
I think saying aiming for only MVP isn't great marketing. It sound like we're saying that will do the bare minimum rather that what I think was intended.

Reply:
I like your one, I was trying to call back to the Mural too directly?